submodules/device/device.js

```diff
@@ -402,52 +402,72 @@ define(function(require) {
 		},
 
 		deviceGetValidationByDeviceType: function(deviceType) {
-			var validation = {
-				ata: {
-					'sip.ip': {
-						required: true,
-						ipv4: true
-					}
-				},
-				sip_uri: {},
-				sip_device: {
-					'mac_address': { mac: true },
-					'sip_expire_seconds': {	digits: true },
-					'sip.ip': {
-						ipv4: true,
-						required: true
+			var self = this,
+				i18n = self.i18n.active(),
+				validation = {
```

Comment:
Yes. Also to avoid multiple calls to `self.i18n.active()`, and to make it easy to merge the extra rules and messages for some device types into the validation options.